.env.development

```diff
@@ -1,3 +1,4 @@
-API_SERVICE_BACKEND_URL_OPTIONS={"uk":"https://dev.trade-tariff.service.gov.uk","xi":"https://dev.trade-tariff.service.gov.uk/xi"}
+API_SERVICE_BACKEND_URL_OPTIONS={"uk":"http://localhost:3000/","xi":"http://localhost:3000/"}
```

Comment:
Implication is that we now need to run the backend locally on port 3000 with reasonably up-to-date data to roll the duty calculator application